### PR TITLE
Timer controls 'Cancel', 'pause', and 'reset'

### PR DESCRIPTION
**Pull request by Simpcyclassy**

### CODE REVIEW

File: extension/intents/timer/timer.toml [hunk 1]
```diff
@@ -11,40 +11,55 @@ phrase = "Set timer for 2 hours and 5 minutes"
 [timer.close]
 description = "Close timer"
 match = """
+<<<<<<< HEAD
```
Comment (ianb) [inline extension/intents/timer/timer.toml:14]: Looks like you added some conflict markers

**Comment by ianb** [comment]
@awallin this would change "cancel" to stop a timer, if there is a timer. I'm not sure we want to do that, but I'm not sure; thoughts?

If it does cancel the timer, then I think we'd want it to show a confirmation message (and so it also shouldn't close immediately).

File: extension/intents/timer/timer.toml (hunk 1 above)
Comment (Simpcyclassy) [reply extension/intents/timer/timer.toml:14]: My bad. Will fix that now

**Comment by awallin** [comment]
`Cancel` should stop the timer. And yes, we should confirm that with the checkmark with "Timer Canceled" below.

**Comment by Simpcyclassy** [comment]
@ianb may I get pointers on how to add this confirmation message. I tried a `context.displayText()` and it gave no result.

Secondly for the pause aspect, if I were to use same implementation I used for cancel and there are a couple of matching intents for `music.pause` it would mean that an utterance like `pause music` will pause timer, I do not think that will be ideal.

**Pull request merged**